parlai/agents/drqa/utils.py

```diff
@@ -139,7 +139,7 @@ def batchify(batch, null=0, cuda=False):
     # Batch documents and features
     max_length = max([d.size(0) for d in docs])
     x1 = torch.LongTensor(len(docs), max_length).fill_(null)
-    x1_mask = torch.ByteTensor(len(docs), max_length).fill_(1)
+    x1_mask = torch.BoolTensor(len(docs), max_length).fill_(1)
```

Comment:
drqa changes were because pytorch 1.4 was throwing a fit when I was trying to debug why it was broken by my changes